pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -646,7 +649,7 @@ func (nv *NonVotingSpec) SetDefaults(cr *PerconaServerMongoDB, rs *ReplsetSpec)
 			Command: []string{"/data/db/mongodb-healthcheck", "k8s", "liveness"},
 		}
 
-		if !cr.Spec.UnsafeConf {
+		if !cr.Spec.UnsafeConf || cr.CompareVersion("1.15.0") < 0 {
```

Comment:
is it correct? before we were adding ssl flags if `UnsafeConf` is false but with these changes we'll add flags if crVersion is < 1.15 no matter unsafe flag value

Reply:
Before the PR, we had this code without any checks:
```
 Command: []string{
				"/data/db/mongodb-healthcheck",
				"k8s",
				"liveness",
				"--ssl", "--sslInsecure",
				"--sslCAFile", "/etc/mongodb-ssl/ca.crt",
				"--sslPEMKeyFile", "/tmp/tls.pem",
			},
```
We should add these flags to crs with < 1.15.0 versions to maintain the old behavior